cdc/sink/cdclog/s3.go

```diff
@@ -289,6 +284,26 @@ func (s *s3Sink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) error {
 	if err != nil {
 		return cerror.WrapError(cerror.ErrS3SinkStorageAPI, err)
 	}
+
+	// only reboot and size = 0 should we add version to s3
+	withVersion := firstCreated && size == 0
```

Comment:
If `size > maxDDLFlushSize`, should the `withVersion` be true?

Reply:
Oh, you are right, maxDDLFlushSize should also be considered